app/controllers/candidate_api/candidates_controller.rb

```diff
@@ -35,23 +35,25 @@ def serialized_candidates
                             .order('application_forms.created_at DESC')
 
       candidates.map do |candidate|
+        current_application = candidate.application_forms.order(:created_at).last
```

Comment:
If we don't grab the latest application form and instead use current_application we create a new application form for candidates who have not signed in. This would mean when a candidate creates an account and git pull from us, we would artificially move a candidate past the not signed in